### PR TITLE
Add feature_disable_call_per_sender_encryption documentation (in labs.md)

### PR DESCRIPTION
**Pull request by toger5**
## Checklist

-   [x] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/vector-im/element-web/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## ✨ Features
 * Add feature_disable_call_per_sender_encryption documentation (in labs.md) ([\#26548](https://github.com/vector-im/element-web/pull/26548)). Contributed by @toger5.

### CODE REVIEW

File: docs/labs.md [hunk 1]
```diff
@@ -120,6 +120,15 @@ This feature allows users to place and join native [MSC3401](https://github.com/
 
 If you're enabling this at the deployment level, you may also want to reference the docs for the `element_call` config section.
 
+## Disable per-sender encryption for Element Call (`feature_disable_call_per_sender_encryption`)
+
+The default for embedded Element Call in Element Web is per participant sender key encryption.
```
Comment (richvdh) [inline docs/labs.md:125]: Nit:

```suggestion
The default for embedded Element Call in Element Web is per-participant sender key encryption. 
```

Comment (richvdh) [reply docs/labs.md:125]: ... but only in encrypted rooms? https://github.com/matrix-org/matrix-react-sdk/pull/11837/files#diff-aa8e05e46de65e6a368e1898db9dd420a41e7001bd470febc5212704add0bdeeR664

Comment (richvdh) [reply docs/labs.md:125]: Also, what is "sender key encryption"?

**Review by richvdh — COMMENTED** [review]
generally looks good but I think the wording could do with clarification

File: docs/labs.md (hunk 1 above)
Comment (toger5) [reply docs/labs.md]: > Also, what is "sender key encryption"?

Each sender has its own different key. I think you are right that we can just go with per-participant encryption.

**Comment by richvdh** [comment]
yes agreed

File: docs/labs.md [hunk 2]
```diff
@@ -120,6 +120,15 @@ This feature allows users to place and join native [MSC3401](https://github.com/
...
 If you're enabling this at the deployment level, you may also want to reference the docs for the `element_call` config section.
 
+## Disable per-sender encryption for Element Call (`feature_disable_call_per_sender_encryption`)
+
+The default for embedded Element Call in Element Web is per-participant encryption.
+This labs flag disables encryption for embedded Element Call in encrypted rooms.
+
+Under the hood this stops Element Web from adding the `perParticipantE2EE` flag for the Element Call widget url.
+
+This is useful while we experiment with encryption and to make calling compatible with platforms that don't use encryption it.
```
Comment (aceArt-GmbH) [inline docs/labs.md:130]:
```suggestion
This is useful while we experiment with encryption and to make calling compatible with platforms that don't use encryption yet.
```
Typo?

**Review by Aziz-4980 — COMMENTED** [review]
looks good tp me

**Review by richvdh — APPROVED** [review]
lgtm

**Pull request merged**